scheduler/pkg/kafka/topics.go

```diff
@@ -50,51 +50,54 @@ func NewTopicNamer(namespace string, topicPrefix string) *TopicNamer {
 	}
 	return &TopicNamer{
 		namespace:   namespace,
-		topicPrefix: topicPrefix,
+		topicPrefix: fmt.Sprintf("%s.", topicPrefix),
 	}
 }
 
 func (tn *TopicNamer) GetModelErrorTopic() string {
-	return fmt.Sprintf("%s.%s.%s.%s", tn.topicPrefix, tn.namespace, errorsTopic, errorsSuffix)
+	return fmt.Sprintf("%s%s.%s.%s", tn.topicPrefix, tn.namespace, errorsTopic, errorsSuffix)
 }
 
 func (tn *TopicNamer) GetKafkaModelTopicRegex() string {
-	return fmt.Sprintf("^%s.%s.%s.*.%s", tn.topicPrefix, tn.namespace, modelTopic, inputsSuffix)
+	return fmt.Sprintf("^%s%s.%s.*.%s", tn.topicPrefix, tn.namespace, modelTopic, inputsSuffix)
```

Comment:
maybe add a note that `topicPrefix` as a dot suffix?

Reply:
actions:
 - Helper function for joining topic names